Userland/Libraries/LibCore/MappedFile.cpp

```diff
@@ -14,35 +15,51 @@
 
 namespace Core {
 
-ErrorOr<NonnullRefPtr<MappedFile>> MappedFile::map(StringView path)
+ErrorOr<NonnullOwnPtr<MappedFile>> MappedFile::map(StringView path, OpenMode mode)
```

Comment:
The NNRP -> NNOP change should have been in the previous commit, since we already removed the refcounting.